scripts/vm/hypervisor/kvm/kvmheartbeat.sh

```diff
@@ -155,10 +155,10 @@ then
   exit 0
 elif [ "$cflag" == "1" ]
 then
-  /usr/bin/logger -t heartbeat "kvmheartbeat.sh rebooted system because it was unable to write the heartbeat to the storage."
+  /usr/bin/logger -t heartbeat "kvmheartbeat.sh stopped cloudstack-agent because it was unable to write the heartbeat to the storage."
   sync &
   sleep 5
-  echo b > /proc/sysrq-trigger
+  service cloudstack-agent stop
```

Comment:
Please start a discussion on dev@, this change to simply stop the agent may not be acceptable for all cases.

Reply:
Instead, what about creating a global parameter to determine this behavior? Then, the administrator would be able to change this easily.

Reply:
Good idea @rafaelweingartner, perhaps a cluster scope global parameter. However, such a setting need to be propagated when agent/client connects to mgmt server (say via ReadyCommand) or overridable in agent.properties file.

Reply:
Yes, the externalization for a global settings would need more work, but I think it is worth it.
We could help/guide @Slair1 if he does not where to start then.

Reply:
Maybe you can sent the PR in two parts, the first half that increases the heartbeat retry threshold to `5` LGTM. The second part could be the configuration fix, global setting or setting in agent.properties to decide whether to reboot kvm host or stop agent.